buildSrc/src/main/kotlin/software/aws/toolkits/gradle/resources/ValidateMessages.kt

```diff
@@ -35,18 +38,20 @@ open class ValidateMessages : DefaultTask() {
                 fileLines
                     // filter out blank lines and comments
                     .filter { it.isNotBlank() && it.trim().firstOrNull() != '#' }
-                    .mapNotNull {
+                    .mapIndexed { lineNumber, it ->
                         if (it.contains("=")) {
                             it
                         } else {
-                            logger.warn(""""$filePath contains invalid message missing a '=': "$it"""")
+                            logger.error(""""$filePath:${lineNumber + COPYRIGHT_HEADER_LINES} contains invalid message missing a '=': "$it"""")
```

Comment:
nice the lack of line numbers was super annoying. as a future enhancement can we have a task that autosorts?

Reply:
Yeah that would be good